api/src/main/resources/workbench-api.yaml

```diff
@@ -3884,7 +3884,7 @@ definitions:
         type: string
       approved:
         type: boolean
-        default: 
+        default:
```

Comment:
I didn't make any change here, but I think IntelliJ just removed the trailing whitespace automatically (here and below). I'm not sure how to go about removing these changes if I'm using IJ for editing...